criu/uffd.c

```diff
@@ -269,7 +269,11 @@ int uffd_open(int flags, unsigned long *features)
 
 	uffd = syscall(SYS_userfaultfd, flags);
 	if (uffd == -1) {
-		pr_info("Lazy pages are not available: %s\n", strerror(errno));
```

Comment:
Hello @angieenee, please rearrange your branch so that you have a single commit with a descriptive change log on top of criu-dev.
You can either create a new branch and add the change there or use `git rebase --interactive` to modify the history of your branch